hokusai/build.yml

```diff
@@ -1,9 +1,10 @@
 ---
-version: "2"
+version: "3.8"
 services:
   horizon:
+    image: "hokusai_horizon:${BUILD_TAG:-latest}"
     build:
-      context: ../
+      target: "${BUILD_TARGET}"
```

Comment:
This triggers a warning, still it works perfectly. In the future we may want to have a documented default target.